domain-models-maven-plugin/src/main/java/org/folio/rest/tools/utils/SchemaDereferencer.java

```diff
@@ -106,4 +106,8 @@ static String toFileUri(Path basePathFile, String file) {
     Path absolutePath = basePathFile.resolveSibling(file).normalize();
     return absolutePath.toUri().toString();
   }
+
+  private static boolean hasSelfReference(String refValue) {
+    return refValue.contains("#");
+  }
```

Comment:
Either check for first character to make this conform to the method name ; or rename the method to say hasUriFragment in which the method does what it says. In both cases, things should work. The hasUriScheme would also return true if fragment in a URL.

Reply:
I prefer the former.